x-pack/plugins/alerting/server/alert/alert.test.ts

```diff
@@ -0,0 +1,488 @@
+/*
```

Comment:
Github did not recognize this renaming from `alert_instance/alert_instance.test.ts` to `alert/alert.test.ts`. No changes to the tests, just `AlertInstance` to `Alert`